WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/data/db/WatchFaceDatabase.kt

```diff
@@ -31,7 +31,7 @@ private const val DATABASE_NAME = "analog_watch_face_database"
         AnalogWatchFaceEntity::class,
         WatchFaceColorStyleEntity::class,
         WatchFaceArmDimensionsEntity::class],
-    version = 1,
+    version = 3,
```

Comment:
you should be able to move to 2. 
I guess you had to move to 3 because you made changes to different tables and in between you ran the app, but users of this sample should just have version 1 installed

Reply:
That's exactly what happened (was working on several version before PR).

Changed.